README.md

```diff
@@ -102,7 +102,47 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Vdk-core and CLI
+
+```
+pip install vdk-core
```

Comment:
do we not just recommend installing quickstart-vdk? 
it will pull this in.

Reply:
Yes `vdk-core` nobody should install . We should have different distributions (with collection of plugins inside) for difference use cases. And quickstart-vdk is for one to get stated for first time with VDK.

Reply:
Ah, nvm, the problem was the `~/.vdk.internal` config folder. Not sure if it's documented anywhere, but it's super annoying. It works as expected now. I'll change the description.

Reply:
what was the problem? Please open an issue for that. If it was annoying for you it will be annoying for other people.